docs/topics/mta-cli-run.adoc

```diff
@@ -0,0 +1,133 @@
+
++
+[source,terminal,subs="attributes+"]
+----
+$ ./{mta-cli} analyze --input /path/to/jee-example-app-1.0.0.ear \
+    --output /path/to/output --source weblogic --target eap6 \
+----
++
+* `--input`: The application to be evaluated.
+* `--output`: The output directory for the generated reports.
+* `--source`: The source technology for the application migration.
+
+. Access the report.
+
+[id="command-examples_{context}"]
+== {ProductShortName} command examples
+
+[discrete]
+=== Running {ProductShortName} on an application archive
+
+The following command analyzes the `com.acme` and `org.apache` packages of the link:https://github.com/windup/windup/blob/master/test-files/jee-example-app-1.0.0.ear[jee-example-app-1.0.0.ear] example EAR archive for migrating from JBoss EAP 5 to JBoss EAP 7:
+
+[source,terminal,subs="attributes+"]
+----
+$ <{ProductShortName}_HOME>/{mta-cli} analyze \
+    --input /path/to/jee-example-app-1.0.0.ear \
+    --output /path/to/report-output/ --source eap5 --target eap7 \
+----
+[]
+
+[discrete]
+=== Running {ProductShortName} on source code
+
+The following command analyzes the `org.jboss.seam` packages of the link:https://github.com/windup/windup/tree/master/test-files/seam-booking-5.2[seam-booking-5.2] example source code for migrating to JBoss EAP 6.
+
+[source,terminal,subs="attributes+"]
+----
+
+$ <{ProductShortName}_HOME>/bin/{mta-cli} analyze --mode source-only --input /path/to/seam-booking-5.2/ \
+    --output /path/to/report-output/ --target eap6 --packages org.jboss.seam
+----
+[]
+[discrete]
+=== Running cloud-readiness rules
+
+The following command analyzes the `com.acme` and `org.apache` packages of the link:https://github.com/windup/windup/blob/master/test-files/jee-example-app-1.0.0.ear[jee-example-app-1.0.0.ear] example EAR archive for migrating to JBoss EAP 7. It also evaluates for cloud readiness:
+
+[source,terminal,subs="attributes+"]
+----
+$ <{ProductShortName}_HOME>/bin/{mta-cli} analyze --input /path/to/jee-example-app-1.0.0.ear \
+    --output /path/to/report-output/ \
+    --target eap7
+----
+[]
+////
+[discrete]
+=== Overriding {ProductShortName} properties
+
+To override the default _Fernflower_ decompiler, pass the `-Dwindup.decompiler` argument on the command line. For example, to use the _Procyon_ decompiler, use the following syntax:
+
+[source,terminal,subs="attributes+"]
+----
+$ <{ProductShortName}_HOME>/{mta-cli} -Dwindup.decompiler=procyon \
+    --input <INPUT_ARCHIVE_OR_DIRECTORY> --output <OUTPUT_REPORT_DIRECTORY> \
+----
+////
+////
+[id="cli-bash-completion_{context}"]
+== {ProductShortName} {CLINameTitle} Bash completion
+
+The {ProductShortName} {CLIName} provides an option to enable Bash completion for Linux systems, allowing the {ProductShortName} command-line arguments to be auto completed by pressing the Tab key when entering the commands. For instance, when Bash completion is enabled, entering the following displays a list of available arguments.
```

Comment:
This too needs to be removed.

Reply:
Commented out
![image](https://github.com/windup/windup-documentation/assets/106804821/44b0b9c5-e7cd-4a71-a732-84ad85f62101)